code/game/machinery/GBPvendor.dm

```diff
@@ -0,0 +1,125 @@
+
+/obj/machinery/GBP_vendor
+	name = "engineering point redemption"
+	desc = "Who's a good boy?"
+	icon = 'icons/obj/vending.dmi'
+	icon_state = "liberationstation"
+	density = 1
+	anchored = 1
+	var/GBP = 0
+	var/GBPspent = 0
+	var/GBPearned = 0
+	var/list/prize_list = list( //if you add something to this, please, for the love of god, use tabs and not spaces.

```

Comment:
How do I make it static?

Also I tried to put it in Initialize and I got an error.